ckanext/xloader/jobs.py

```diff
@@ -178,8 +178,8 @@ def xloader_data_into_datastore_(input, job_dict, logger):
     logger.info('Express Load starting: %s', resource_ckan_url)
 
     # check if the resource url_type is a datastore
-    if resource.get('url_type') == 'datastore':
-        logger.info('Ignoring resource - url_type=datastore - dump files are '
+    if resource.get('url_type') in h.datastore_rw_resource_url_types():
```

Comment:
Needs backwards support for for 2.10.x since datastore_rw_resource_url_types only exists in 2.11.

https://github.com/ckan/ckan/blob/ckan-2.11.1/ckanext/datastore/helpers.py#L242 (exists since May 2023)
https://github.com/ckan/ckan/blob/ckan-2.10.6/ckanext/datastore/helpers.py (does not include 'datastore_rw_resource_url_types'  )

i.e.
```
if hasattr(h, "datastore_rw_resource_url_types"):
             datastore_rw_resource_url_types = h.datastore_rw_resource_url_types()
else:
             #fallback for 2.10.x or older.
             datastore_rw_resource_url_types = ['datastore']
           
if resource.get('url_type') in datastore_rw_resource_url_types :
```